scripts/ci/check-dependencies-versions-match.mjs

```diff
@@ -162,14 +180,19 @@ function getExpectedDepsVersions( packageJsons, isCkeditor5Package ) {
  * @return {String}
  */
 function getNewestVersion( packageName, newVersion = '0.0.0', currentMaxVersion = '0.0.0' ) {
-	if ( !semver.valid( newVersion ) ) {
-		const versions = getVersionsList( packageName );
-		const newMaxVersion = semver.maxSatisfying( versions, newVersion );
+	if ( versionExceptions[ packageName ] ) {
+		return newVersion;
+	}
 
-		return semver.gt( newMaxVersion, currentMaxVersion ) ? newMaxVersion : currentMaxVersion;
+	if ( semver.valid( newVersion ) ) {
+		return semver.gt( newVersion, currentMaxVersion ) ? newVersion : currentMaxVersion;
 	}
 
-	return semver.gt( newVersion, currentMaxVersion ) ? newVersion : currentMaxVersion;
+	const versions = getVersionsList( packageName );
+	const newMaxVersion = semver.maxSatisfying( versions, newVersion );
+	const range = versionExceptions[ packageName ] || '';
```

Comment:
It seems that in this line `versionExceptions[ packageName ]` is always `undefined`, because otherwise it would go inside the `if ( versionExceptions[ packageName ] )` condition above.

Reply:
Good catch, I changed the logic and forgot to update this part.